src/libraries/System.ComponentModel.TypeConverter/src/Resources/Strings.resx

```diff
@@ -170,15 +171,15 @@
   </data>
   <data name="CultureInfoConverterInvalidCulture" xml:space="preserve">
     <value>The {0} culture cannot be converted to a CultureInfo object on this computer.</value>
-  </data> 
+  </data>
   <data name="ErrorInvalidServiceInstance" xml:space="preserve">
     <value>The service instance must derive from or implement {0}.</value>
   </data>
   <data name="ErrorServiceExists" xml:space="preserve">
     <value>The service {0} already exists in the service container.</value>
   </data>
   <data name="InvalidArgumentValue" xml:space="preserve">
-    <value>Value of '{1}' is not valid for '{0}'.</value>
+    <value>Value of '{0}' cannot be empty.</value>
```

Comment:
Two different locations throw exceptions with this message.  https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.Drawing.Common/src/System/Drawing/StringFormat.cs#L257-L260 doesn't make sense with this change.  ... of course, it also doesn't make sense with the original (since it doesn't pass enough format arguments).

Either this string needs to work for both sites, or it should be split into two strings.

Reply:
(nevermind, that was in a different library... I mis-clicked)